gems/aws-sdk-core/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 Unreleased Changes
 ------------------
+* Issue - Fix ordering of plugin options for performance optimization (#3002).
```

Comment:
I'd be slightly more specific here - and say something like "Performance optimization:  ensure presence and order of instance variables in PluginOptions"